server/src/models/__mocks__/index.js

```diff
@@ -26,9 +23,42 @@ const User = {
   findOrCreate: jest.fn(() => Promise.resolve(true)),
 };
 
+const Measurable = {
+  findOne: jest.fn(() => Promise.resolve(true)),
```

Comment:
This is not very dry. Can you at either make one fn for ``jest.fn(() => Promise.resolve(true))``, or go further and have a function to generate these?

Reply:
No, it is not a principle of DRY when you use auto-generation function in THIS case. We must call jest.fn in each case where we want to get Jest Mock Object. Each of these objects has own state.

Reply:
It should be fine with mocks this way because it's clear what a mock returns. And we should keep them as different objects with it's own context (calls count, arguments passed).

Reply:
Ok, that works.